Formula/telegram-cli.rb

```diff
@@ -0,0 +1,61 @@
+class TelegramCli < Formula
+  desc "Command-line interface for Telegram."
+  homepage "https://github.com/vysheng/tg"
+  # Use the tag instead of the tarball to get submodules
+  url "https://github.com/vysheng/tg.git",
+      :tag => "1.3.1",
+      :revision => "5935c97ed05b90015418b5208b7beeca15a6043c"
+  head "https://github.com/vysheng/tg.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "readline"
+  depends_on "libevent"
+  depends_on "openssl"
+  depends_on "libconfig"
+  depends_on "jansson"
+  depends_on "lua" => :recommended
+  depends_on "python" => :recommended
+
+  # Look for the configuration file under /usr/local/etc rather than /etc on OS X.
```

Comment:
Okay, a PR is submitted, and the comment in the formula links to it now. Phew, a lot of work for a little patch! :) Hope all is in order to merge now.